src/nft/components/explore/CarouselCard.tsx

```diff
@@ -291,7 +291,7 @@ const DEFAULT_TABLE_ELEMENTS = 12
 const LoadingTable = () => {
   return (
     <>
-      {[...Array(DEFAULT_TABLE_ELEMENTS)].map((index) => (
+      {Array.from({ length: DEFAULT_TABLE_ELEMENTS }, (_, i) => i + 1).map((_, index) => (
```

Comment:
Was CarouselCard also getting a unique key warning? why was this added?

Reply:
Yes, CarouselCard also getting a unique key warning.
The `[...Array(DEFAULT_TABLE_ELEMENTS)]` code return an array of undefined, instead `Array.from({ length: DEFAULT_TABLE_ELEMENTS }, (_, i) => i + 1)` return an array of number element